research/conditional/utils/conditional_trainer.py

```diff
@@ -130,6 +137,25 @@ def _after_train_operations(self):
         if self.is_logging_process:
             self.logger.exit_job_metadata(self.current_step)
 
+        # Above - end of job training operations area (not model trainig (one config (even diverted from a grid config) can results in multiple models with trapezoidal lr checpotin manager))
+        if self.current_step >= n_steps:
+            # Below - end of model training operations area
```

Comment:
area -> _following lines_ or _the rest of this function_, etc.

Reply:
changed